joomla/joomla-cms
# [3.7] Hathor postinstall

### Summary of Changes

Based on the feedback from the new PLT we are adding a new postinstll message for any users that are using hathor inorder to tell them that hathor is not getting new features.

### Testing Instructions

- Apply this path
- run the SQL Query (for your database system)
- set template to hathor
- confirm the message + button is there
- hit the button
- confirm the template is now isis and there is a message about the change
- swich back to hathor
- confirm that you can hide the message
- please start a new install based on https://github.com/zero-24/joomla-cms/archive/hathor_postinstall.zip
- and confirm the same behavior (enable and disable of hathor) there.

### Expected result

- The button switch the template style to isis
- the message get only displayed on hathor

### Actual result

There is no message 😄 

### Documentation Changes Required

None as hathor is already removed in 4.0.

### Other things.

1. Please review the new language strings as there are far away from optimal.
2. please review & test the new install and updates for postgresql and mssql.
